docs/contributing.rst

```diff
@@ -257,5 +257,12 @@ without warnings. You can do that locally with::
 Review process
 --------------
 
-Pull requests will be reviewed by a project maintainer. All changes to *sgkit* require
-approval by at least one maintainer.
+All pull requests require pre-existing issue. If you plan to submit a PR,
+please create an issue first to describe and discuss your idea/plan/problem.
+
+Pull requests will be reviewed by a project maintainer. All changes to *sgkit*
+require approval by at least one maintainer.
+
+We use `mergify <https://mergify.io/>`_ to automate PR flow, should you like
```

Comment:
Actually, come to think of it, I don't think this is the right protocol. The person who opens the PR shouldn't decide whether something should be automerged, it should be an admin who makes this decision. So, if one admin reviews the PR and thinks it's OK, they mark it automerge. If they think other admins should have a look, they don't.

Reply:
I think it'll really confuse more casual contributors too to have to understand whether they should mark as auto-merge or not.

Reply:
Sure - that sound fine to me as well.

Reply:
@jeromekelleher updated.